### ISSUE TITLE
v1.2.0 when using ic-on-before-trigger - intercooler.min.js:1104 Uncaught ReferenceError: onError is not defined

### ISSUE THREAD

**Issue by dlydiard** [LeadDyno/intercooler-js#225]
i changed the line to 
 ```
var onError = closestAttrValue(elt, 'ic-on-error');
if (onError) {
     ...
}
```
and it fixed the problem. i didn't get this error in 1.0.3, seems like onError is not in the scope of handleTriggerOn()

**Comment by carsongross** [comment]
Thanks for the report, I have a fix (and a unit test :/) and will release a 1.2.1 soon.

**Comment by carsongross** [comment]
fixed in the 1.2.1 release
